packages/webapp/src/components/Finances/DateRangeSelector/styles.module.scss

```diff
@@ -1,51 +1,3 @@
-.pullLeft {
```

Comment:
These styles seemed unused, @SayakaOno double check please 🙏

Reply:
Confirmed, I believe you are right!